classes/item.rb

```diff
@@ -0,0 +1,20 @@
+class Item
+  attr_accessor :id, :genre, :author, :label, :publish_date, :archive
+
+  def initialize(genre:, author:, label:, publish_date:)
+    @id = Random.rand(1..10_000)
+    @genre = genre
+    @author = author
+    @label = label
+    @publish_date = publish_date
+    @archive = false
+  end
+
+  def move_to_archive
+    @archive = true if can_be_archived?
+  end
+
+  def can_be_archived?
+    @publish_date > 10
+  end
+end
```

Comment:
- [ ] Currently, there is no test case for the `Item` class. It is required that all public methods in all classes should have unit tests. Please let's create unit tests for these methods `can_be_archived?` and ` move_to_archive` to meet the requirements.